packages/browser/__tests__/login/oidc/ClientRegistrar.spec.ts

```diff
@@ -168,7 +168,7 @@ describe("ClientRegistrar", () => {
           }
         )
       ).rejects.toThrowError(
-        "Client registration failed: [Error: Dynamic client registration failed: bad stuff that's an error - ]"
+        "Client registration failed: [Error: Dynamic client registration failed: bad stuff that's an error - undefined]"
```

Comment:
I haven't seen the code that throws yet, but our test expecting an error message with 'undefined' in it doesn't 'feel' right...?

Reply:
Fixed in eb47a0c19491f23bfbffe99a06c61634d39300ae